app/controllers/matches_controller.rb

```diff
@@ -0,0 +1,88 @@
+class MatchesController < ApplicationController
```

Comment:
It looks like most of your controller actions are auto generated and probably aren't needed. Can you remove the ones you don't want? Same thing with the other controllers

Reply:
Many of them are auto-generated, but to support the matches and rounds in the Web UI it seems like I need them